sdks/python/apache_beam/io/gcp/tests/bigquery_matcher.py

```diff
@@ -144,7 +144,18 @@ def _matches(self, _):
     self.actual_data = response
 
     # Verify result
-    return sorted(self.expected_data) == sorted(self.actual_data)
+    try:
```

Comment:
Is there a reason we cannot reuse https://github.com/apache/beam/blob/2d5e493abf39ee6fc89831bb0b7ec9fee592b9c5/sdks/python/apache_beam/testing/util.py#L117 here?

Reply:
You are right, it is better to reuse this function. I updated my code